--- iax-0.2.2/src/Makefile.am~	Fri Sep  7 16:48:17 2001
+++ iax-0.2.2/src/Makefile.am	Sun Jun 15 22:12:23 2003
@@ -1,18 +1,11 @@
-CFLAGS += -g -Wall -Wstrict-prototypes -I .
-CFLAGS += -DDEBUG_SUPPORT -DDEBUG_DEFAULT 
-CFLAGS += $(UCFLAGS)
-
-
-pkgdir = $(libdir)
-pkg_LTLIBRARIES=libiax.la
+lib_LTLIBRARIES=libiax.la
 libiax_la_SOURCES = iax.c md5.c 
 EXTRA_DIST = md5.h frame.h iax-client.h iax.h
 
-install:
-	mkdir -p $(includedir)/iax
-	install -m 644 md5.h $(includedir)/iax
-	install -m 644 frame.h $(includedir)/iax
-	install -m 644 iax.h $(includedir)/iax
-	install -m 644 iax-client.h $(includedir)/iax
-	/sbin/ldconfig
+install-data-local:
+	mkdir -p $(DESTDIR)/$(includedir)/iax
+	install -m 644 md5.h $(DESTDIR)/$(includedir)/iax
+	install -m 644 frame.h $(DESTDIR)/$(includedir)/iax
+	install -m 644 iax.h $(DESTDIR)/$(includedir)/iax
+	install -m 644 iax-client.h $(DESTDIR)/$(includedir)/iax
 
